SolutionInfo.cs

```diff
@@ -9,15 +8,13 @@
 [assembly: AssemblyProduct("Basisregisters Vlaanderen")]
 [assembly: AssemblyCopyright("Copyright (c) Vlaamse overheid")]
 [assembly: AssemblyCompany("Vlaamse overheid")]
-
-namespace System;
-
-internal static class AssemblyVersionInformation
-{
-    internal const string AssemblyCompany = "Vlaamse overheid";
-    internal const string AssemblyCopyright = "Copyright (c) Vlaamse overheid";
-    internal const string AssemblyFileVersion = "2.0.0.0";
-    internal const string AssemblyInformationalVersion = "";
-    internal const string AssemblyProduct = "Basisregisters Vlaanderen";
-    internal const string AssemblyVersion = "2.0.0.0";
-}
\ No newline at end of file
+namespace System {
+    internal static class AssemblyVersionInformation {
```

Comment:
please keep your braces on separate lines

Reply:
It's the `build.sh publish` script which changed this. Every time you do a publish this file will change. Can we leave these changes to avoid updates every time? Or the build script needs to be updated to match the formatting rules.

Reply:
ok